mqtt-client-WebSocket/ws-mqtt.html

```diff
@@ -50,10 +53,10 @@ <h2>WebSocket over TLS/SSL</h2>
     const connectUrl = 'wss://broker.emqx.io:8084/mqtt'
 
     const options = {
-      keepalive: 30,
+      keepalive: 60,
       clientId: clientId,
       clean: true,
-      connectTimeout: 5000,
+      connectTimeout: 30 * 1000,
```

Comment:
Why change this to an expression?

Reply:
5s is too short, the expression looks more readable.